src/pages/publication/contributors_tab/components/SortableTable.tsx

```diff
@@ -71,45 +75,41 @@ const SortableItem = SortableElement(
             )}
           </SubHeading>
 
-          {contributor.identity.arpId ? (
-            <>
-              <Field name={`${baseFieldName}.${SpecificContributorFieldNames.CORRESPONDING}`}>
-                {({ field }: FieldProps) => (
-                  <FormControlLabel
-                    control={<Checkbox checked={field.value} {...field} />}
-                    label={t('publication:contributors.corresponding')}
+          <Field name={`${baseFieldName}.${SpecificContributorFieldNames.CORRESPONDING}`}>
+            {({ field }: FieldProps) => (
+              <FormControlLabel
+                control={<Checkbox checked={field.value} {...field} />}
+                label={t('publication:contributors.corresponding')}
+              />
+            )}
+          </Field>
+          <div>
```

Comment:
div ble brukt for styling her, som er litt æsj... men vi må uansett ta en runde med styling her når vi har affiliations klar på contributor, så gjøre minst mulig i denne omgang